includes/class-sensei-learner-profiles.php

```diff
@@ -25,7 +25,7 @@ class Sensei_Learner_Profiles {
 	public function __construct() {
 
 		// Setup learner profile URL base
-		$this->profile_url_base = apply_filters( 'sensei_learner_profiles_url_base', __( 'learner', 'sensei-lms' ) );
+		$this->profile_url_base = apply_filters( 'sensei_learner_profiles_url_base', __( 'student', 'sensei-lms' ) );
```

Comment:
Is it okay that people can get broken URLs here?

Reply:
Good point will check this

Reply:
Hey, so I've checked and it turns out that this was never supposed to be something for translating. This URL is always used with learners, so even before it was wrong. There are two cases where this is used, luckily one was used only to show example URL and this one is setting up this URL somewhere but I think only in the config, since redirection was working as expected when I go to **/learner/username** doesn't matter what is written there.